great_expectations/rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder.py

```diff
@@ -130,11 +133,11 @@ def _build_parameters(
         unique_parameter_values: Set[
             Any
         ] = _get_unique_values_from_nested_collection_of_sets(
-            parameter_value_node["value"]
+            parameter_value_node.value
         )
 
         return (
-            unique_parameter_values,
+            sorted(unique_parameter_values),
```

Comment:
Why is this necessary?

Reply:
@cdkini Getting rid of it (found it to be unnecessary).  Thanks.